feature/card-0055/remove-profile-and-id-from-navbar-website

JSantosMoraes opened the pull request:
# [📝 Card-0055 / Remover Perfil e ID NavBar - Site ORIENTA](https://app.clickup.com/t/86acj4dmd)

## 🎯 Objetivo da Task
O objetivo principal desta task é remover a exibição da foto de perfil e do ID do usuário atualmente presentes na navbar do site, garantindo uma interface mais limpa e alinhada ao conceito de minimalismo adotado pelo projeto.

## 📂 Arquivos Removidos
- _website/img/perfil.jpg_

## 🔄️ Arquivos Modificados
1. _Todos os arquivos *.html*_
   - O perfil ao lado da navbar foi removido.
2. _style.css_
   - Alinhamento horizontal da navbar.
   - Ajuste da largura da navbar.

JSantosMoraes committed:
💄remove: remove profile section from search modal

As informações de perfil foram removidas da parte de cima das páginas.

JSantosMoraes commented:
Aguardando a review de vocês @felipebarbosa24, @MateusPereira1 e @Killao1.

felipebarbosa24 reviewed (changes requested):
Olá @JSantosMoraes, revisando a task notei a seguinte melhoria:

1. Que nem citado anteriormente, ao remover o perfil e o ID a navbar estica para preencher o espaço não alocado:

**Main:**
<img width="1901" height="173" alt="image" src="https://github.com/user-attachments/assets/5578931f-713e-4909-bf21-268a3347c196" />

**Sua Branch:**
<img width="1899" height="167" alt="image" src="https://github.com/user-attachments/assets/42efbbec-730d-4816-9225-340e0cff7c0e" />

Em minha visão a forma ideal é igual como está na Main. Porque a logo fica mais a esquerda e os elementos ficam bem alinhadinhos, recomendo somente essas alterações, deixar igual a **main**, só que sem o Perfil e ID.

JSantosMoraes commented:
> Olá @JSantosMoraes, revisando a task notei a seguinte melhoria:
> 
> 1. Que nem citado anteriormente, ao remover o perfil e o ID a navbar estica para preencher o espaço não alocado:
> 
> **Main:** <img alt="image" width="1901" height="173" src="https://private-user-images.githubusercontent.com/184983723/503021071-5578931f-713e-4909-bf21-268a3347c196.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjA5MTcwODcsIm5iZiI6MTc2MDkxNjc4NywicGF0aCI6Ii8xODQ5ODM3MjMvNTAzMDIxMDcxLTU1Nzg5MzFmLTcxM2UtNDkwOS1iZjIxLTI2OGEzMzQ3YzE5Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAxOVQyMzMzMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZDUxM2QyOGNiNTRlNmMzN2ZhOTM3Mjg3NTgzZjRjNWI0YjA5NWVlYjY0MGIxOTYzOTg2MDEwZDlmZWFjNzAzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.XCOFK_tNLcuOPZ5rDDhAZMolqETwH5tzkWY938lg9ys">
> 
> **Sua Branch:** <img alt="image" width="1899" height="167" src="https://private-user-images.githubusercontent.com/184983723/503021016-42efbbec-730d-4816-9225-340e0cff7c0e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjA5MTcwODcsIm5iZiI6MTc2MDkxNjc4NywicGF0aCI6Ii8xODQ5ODM3MjMvNTAzMDIxMDE2LTQyZWZiYmVjLTczMGQtNDgxNi05MjI1LTM0MGUwY2ZmN2MwZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAxOVQyMzMzMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYzQzNjBlMjljZTg3NDAzN2UxYmU0ZjdjNDQ0MTlmNzA4MTM3ZGI2ZTJmN2M4MzM5MmE1NmFiNWQ0ZGY0MjQ0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hCUPmK51BTm__eMiOS3-rOoyz68Btfb61cOoVlXxh7M">
> 
> Em minha visão a forma ideal é igual como está na Main. Porque a logo fica mais a esquerda e os elementos ficam bem alinhadinhos, recomendo somente essas alterações, deixar igual a **main**, só que sem o Perfil e ID.

Na verdade a navbar não esticou, mantive em um tamanho parecido com o original. O que aconteceu foi que, após isso, adicionei margin: auto; para que ficasse centralizada, compensando a falta da seção de perfil. Observe que a logo do ORIENTA também ficou mais ao centro.

Será que fica melhor assim centralizada, ou a navbar mais alinhada à esquerda, parecida com o main?

@felipebarbosa24

JSantosMoraes committed:
Merge branch 'main' into feature/card-0055/remove-profile-and-id-from-navbar-website

felipebarbosa24 commented:
> Olá @JSantosMoraes, revisando a task notei a seguinte melhoria:
> 
> 1. Que nem citado anteriormente, ao remover o perfil e o ID a navbar estica para preencher o espaço não alocado:
> 
> **Main:** <img alt="image" width="1901" height="173" src="https://private-user-images.githubusercontent.com/184983723/503021071-5578931f-713e-4909-bf21-268a3347c196.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjA5MjcyODAsIm5iZiI6MTc2MDkyNjk4MCwicGF0aCI6Ii8xODQ5ODM3MjMvNTAzMDIxMDcxLTU1Nzg5MzFmLTcxM2UtNDkwOS1iZjIxLTI2OGEzMzQ3YzE5Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyMFQwMjIzMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMWZiYmM4ZWU1YTUyYWY4ZTc1MmVkZThlNTZmMTkzZmRkOTk0MjdkZWY1ZDk1MTQ1MDI3NzY5YmY5M2RhM2YwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.kG-wR0QC0nWoKssAe-CLDoPCNWT0W_p73-2Rb4r9NFE">
> 
> **Sua Branch:** <img alt="image" width="1899" height="167" src="https://private-user-images.githubusercontent.com/184983723/503021016-42efbbec-730d-4816-9225-340e0cff7c0e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjA5MjcyODAsIm5iZiI6MTc2MDkyNjk4MCwicGF0aCI6Ii8xODQ5ODM3MjMvNTAzMDIxMDE2LTQyZWZiYmVjLTczMGQtNDgxNi05MjI1LTM0MGUwY2ZmN2MwZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyMFQwMjIzMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMTRlNGM5NGYwOTcxZTE4ZWQyY2ZlZjliMTQyNmY3M2RmY2YxM2M0ODdiMmUyYjc3ZmVhYWRmOTljMTM0MTM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hrCIIp5cZDSy1wzpnRtNoEMebZdfP752KWni7fZhJ78">
> 
> Em minha visão a forma ideal é igual como está na Main. Porque a logo fica mais a esquerda e os elementos ficam bem alinhadinhos, recomendo somente essas alterações, deixar igual a **main**, só que sem o Perfil e ID.



> > Olá @JSantosMoraes, revisando a task notei a seguinte melhoria:
> > 
> > 1. Que nem citado anteriormente, ao remover o perfil e o ID a navbar estica para preencher o espaço não alocado:
> > 
> > **Main:** <img alt="image" width="1901" height="173" src="https://private-user-images.githubusercontent.com/184983723/503021071-5578931f-713e-4909-bf21-268a3347c196.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjA5MTcwODcsIm5iZiI6MTc2MDkxNjc4NywicGF0aCI6Ii8xODQ5ODM3MjMvNTAzMDIxMDcxLTU1Nzg5MzFmLTcxM2UtNDkwOS1iZjIxLTI2OGEzMzQ3YzE5Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAxOVQyMzMzMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZDUxM2QyOGNiNTRlNmMzN2ZhOTM3Mjg3NTgzZjRjNWI0YjA5NWVlYjY0MGIxOTYzOTg2MDEwZDlmZWFjNzAzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.XCOFK_tNLcuOPZ5rDDhAZMolqETwH5tzkWY938lg9ys">
> > **Sua Branch:** <img alt="image" width="1899" height="167" src="https://private-user-images.githubusercontent.com/184983723/503021016-42efbbec-730d-4816-9225-340e0cff7c0e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjA5MTcwODcsIm5iZiI6MTc2MDkxNjc4NywicGF0aCI6Ii8xODQ5ODM3MjMvNTAzMDIxMDE2LTQyZWZiYmVjLTczMGQtNDgxNi05MjI1LTM0MGUwY2ZmN2MwZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAxOVQyMzMzMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYzQzNjBlMjljZTg3NDAzN2UxYmU0ZjdjNDQ0MTlmNzA4MTM3ZGI2ZTJmN2M4MzM5MmE1NmFiNWQ0ZGY0MjQ0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hCUPmK51BTm__eMiOS3-rOoyz68Btfb61cOoVlXxh7M">
> > Em minha visão a forma ideal é igual como está na Main. Porque a logo fica mais a esquerda e os elementos ficam bem alinhadinhos, recomendo somente essas alterações, deixar igual a **main**, só que sem o Perfil e ID.
> 
> Na verdade a navbar não esticou, mantive em um tamanho parecido com o original. O que aconteceu foi que, após isso, adicionei margin: auto; para que ficasse centralizada, compensando a falta da seção de perfil. Observe que a logo do ORIENTA também ficou mais ao centro.
> 
> Será que fica melhor assim centralizada, ou a navbar mais alinhada à esquerda, parecida com o main?
> 
> @felipebarbosa24

Olá @JSantosMoraes.
> Em minha visão a forma ideal é igual como está na Main. Porque a logo fica mais a esquerda e os elementos ficam bem alinhadinhos, recomendo somente essas alterações, deixar igual a **main**, só que sem o Perfil e ID.

Assim como disse, acho que fica melhor de uma forma mais parecida possível com a Main, só que sem o perfil e ID. Resumindo, se na main está mais a esquerda a logo, deixa mais a esquerda, o mesmo serve para todos os elementos da navbar, vai comparando pra deixar idêntico.

JSantosMoraes committed:
💄style: navbar alignment

A navbar agora esta mais proxima com a versão com main, só que sem o perfil ao lado.

JSantosMoraes commented:
Olá @felipebarbosa24, @MateusPereira1 e @Killao1. Acabei de atualizar a navbar para ficar de acordo com o que foi pedido.

felipebarbosa24 reviewed (changes requested):
Olá @JSantosMoraes, ficou top e do jeitinho que eu disse. Parabéns pela task! Vou aqui só solicitar só pra mover um pouquinho mais para direita a navbar, olhando nessa nova perspectiva.
<img width="1898" height="517" alt="Captura de tela 2025-10-21 191238" src="https://github.com/user-attachments/assets/dcef2c7e-9687-42f4-aa93-387506b6e0f0" />

JSantosMoraes committed:
💄style: navbar aligned further to the right

- A navbar foi esticada mais a direita.

JSantosMoraes commented:
Olá @felipebarbosa24, @MateusPereira1 e @Killao1.

 Acabei de melhorar o alinhamento da navbar pra ficar de acordo com o que foi pedido. Aguardando review.

MateusPereira1 reviewed (approved):
olá @JSantosMoraes tá bem legal, não vejo nada o que solicitar a mais, vou aprovar a sua **Task**

felipebarbosa24 reviewed (approved):
Olá @JSantosMoraes estou aprovando seu PR!✅ Parabéns pelo trabalho👏🏻
